code/modules/mob/living/carbon/human/death.dm

```diff
@@ -45,6 +45,5 @@
 
 /mob/living/carbon/proc/Drain()
 	become_husk()
-	disabilities |= NOCLONE

```

Comment:
to elaborate, i swapped most NOCLONE checks for HUSK checks or HUSK plus NOCLONE checks, so NOCLONE is actually forbidding while HUSK can be worked around by upgrading the cloner.